src/lib/libembind.js

```diff
@@ -914,10 +911,16 @@ var LibraryEmbind = {
 #endif
 #if MEMORY64 || CAN_ADDRESS_2GB
       if (signature.includes('p')) {
-        return getDynCaller(signature, rawFunction);
+        return getDynCaller(signature, rawFunction, isAsync);
+      }
+#endif
+      var rtn = getWasmTableEntry(rawFunction);
+#if JSPI
+      if (isAsync) {
```

Comment:
Should we just always use `getDynCaller` maybe?  Or always use it in JSPI mode?

Reply:
I'd like to avoid dynCall since it looks like it adds quite a bit of overhead. It creates a wrapper and then that wrapper calls into getWasmTable entry every time the function is called. In non dyncall mode, the current code just directly calls the function or the the promising wrapper.

Reply:
Fair enough